java/src/test/java/ai/rapids/cudf/TableTest.java

```diff
@@ -7899,6 +7900,54 @@ void testParquetWriteToFileUncompressedNoStats() throws IOException {
     }
   }
 
+  @Test
+  void testParquetWriteWithFieldId() throws IOException {
```

Comment:
There should be a test that we are doing the correct thing when _not_ specfying a field ID to use.

Reply:
Added a not specified `c5` column in the existing test case.